backend/service/k8s/event.go

```diff
@@ -57,9 +56,50 @@ func ProtoForEvent(cluster string, k8sEvent *corev1.Event) *k8sapiv1.Event {
 	}
 }
 
-func (s *svc) ListNamespaceEvents(ctx context.Context, clientset, cluster, namespace string) ([]*k8sapiv1.Event, error) {
-	// TODO: actually implement the function correctly
-	return nil, errors.New("not implemented")
+func convertTypesToFieldSelector(types []k8sapiv1.EventType) string {
```

Comment:
no field selector would query for all event types right? perhaps we can add a code comment

Reply:
👍